WTWSMS/common/nicknames/00_nicknames.txt

```diff
@@ -1925,8 +1949,10 @@ nick_the_wolf = {
 #################################
 nick_the_hammer = {
 	allow = {
-		trait = brave
-		trait = champion
+		OR = {
+			trait = brave
+			trait = champion
+		}
```

Comment:
That was intentional. Using "AND" makes the Nickname essentially impossible to get. Look above it, the other requirements are still there.